docs/source/create_channel/create_channel_test_net.md

```diff
@@ -0,0 +1,444 @@
+
+### Set up the configtxgen tool
+
+Channels are created by building a channel creation transaction and submitting the transaction to the ordering service. The channel creation transaction specifies the initial configuration of the channel and is used by the ordering service to write the channel genesis block. While it is possible to build the channel creation transaction file manually, it is easier to use the [configtxgen](../commands/configtxgen.html) tool. The tool reads the test network `configtx\configtx.yaml` file that defines the configuration of our channel, and then writes the relevant information into the channel creation transaction. When you [installed Fabric](install.html), the `configtxgen` tool was installed in the `fabric-samples\bin` directory for you.
+
+Ensure that you are still operating from the `test-network` directory of your local clone of `fabric-samples` and run this command:
+
+```
+export PATH=${PWD}/../bin:$PATH
+```
+
+Next, before you can use `configtxgen`, you need to the set the `FABRIC_CFG_PATH` environment variable to the location of the test network folder that contains the `configtx.yaml` file. Because we are using the test network, we reference the `configtx` folder:
+```
+export FABRIC_CFG_PATH=${PWD}/configtx
+```
+
+Now verify that you can use the tool by printing the `configtxgen` help text:
+```
+configtxgen --help
+```
+
+### The configtx.yaml file
+
+The `configtx.yaml` file specifies the **channel configuration** of new channels. The information that is required to build the channel configuration is specified in a readable and editable form in the `configtx.yaml` file. The `configtxgen` tool uses the channel profiles that are defined in the `configtx.yaml` file to create the channel configuration and write it to the [protobuf format](https://developers.google.com/protocol-buffers) that can be read by Fabric.
+
+You can find the `configtx.yaml` file that is used to deploy the test network in the `configtx` folder in the `test-network/configtx` directory. The file contains the following information that we will use to create our new channel:
+
+- **Organizations:** The organizations that can become members of your channel. Each organization has a reference to the cryptographic material that is used to build the [channel MSP](../membership/membership.html).
+- **Ordering service:** Which ordering nodes will form the ordering service of the network, and consensus method they will use to agree to a common order of transactions. This section also defines the ordering nodes that are part of the ordering service consenter set. In the test network sample, there is only a single ordering node, but in a production network we recommend **five** ordering nodes to allow for two ordering nodes to go down and still maintain consensus.
+    ```
+    Addresses:
+        - orderer.example.com:7050
```

Comment:
The Addresses entry should be removed from here, and in fact from the test network configtx.yaml file itself.
We now recommend to only specify the OrdererEndpoints in the org section above, you can see it here:
https://github.com/hyperledger/fabric-samples/blob/master/test-network/configtx/configtx.yaml#L44-L45

Reply:
I did not know that. Removed it from the tutorial.